customer-models/src/main/java/org/powertac/customer/evcharger/DemandSampler.java

```diff
@@ -46,10 +48,18 @@
   private static final Logger log = LogManager.getLogger(DemandSampler.class.getSimpleName());
   private MixtureMultivariateNormalDistribution pluginProbability;
   private HashMap<String, MixtureMultivariateNormalDistribution> condHorizonDemandProbabilities = new HashMap<>();
-  private Integer seed;
+  private Long currentSeed;
+  private RandomDataGenerator randomSeedGenerator;
   private XMLConfiguration config;
   private boolean enabled = true;
 
+  void initialize (String model, RandomSeed randomSeed)
+  {
+    this.initialize(model);
+    randomSeedGenerator = new RandomDataGenerator();
```

Comment:
We seed this `RandomDataGenerator` once on initialization with the `org.powertac.common.RandomSeed` passed from EvCharger. Then during each `DemandSampler.sample()` we retrieve the next seed from the `RandomDataGenerator`.

Reply:
I don't see randomSeed being used anywhere.